jenkins/android.jenkinsfile

```diff
@@ -26,16 +26,67 @@ pipeline {
                 script{
                     sh "gclient sync"
                     sh "python ${env.androidBuildNativePath}"
+                    sh "cp -r ${env.andoridWebrtcLibSrcPath}/* ${env.remoteDependeceDebugPath}/libwebrtc"
                     if ("${GIT_BRANCH}" == 'master') { 
                         sh "cp -r ${env.andoridWebrtcLibSrcPath}/* ${env.andoridLibWebrtcPath}"
                     }
                 }
             }
         }
+        stage('startMCU'){
+            agent{
+                node {
+                    label "${env.mcuServer}"
+                }
+            }
+            steps {
+                script{
+                    withEnv(['JENKINS_NODE_COOKIE=dontkill']) {
+                        sh "python ${env.startServerScriptPath} --package-url ${env.mcuPackageBaseDir} --package-name \
+                        ${env.mcuPackageName}  --base-dir ${env.mcuServerBasePath} --git-branch ${GIT_BRANCH} \
+                        --owner open-webrtc-toolkit --repo owt-client-native --commit-id ${GIT_COMMIT} \
+                        --github-script ${env.serverGithubScriptPath} --mode mcu --workspace ${WORKSPACE}"
+                    }
+                }
+            }
+        }
+        stage('startP2PServer'){
+            agent{
+                node {
+                    label "${env.p2pServer}"
+                }
+            }
+            steps{
+                script{
+                    withEnv(['JENKINS_NODE_COOKIE=dontkill']) {
+                        sh "python ${env.startServerScriptPath} --p2p-server-path ${env.p2pServerPath} \
+                        --owner open-webrtc-toolkit --repo owt-client-native --commit-id ${GIT_COMMIT} \
+                        --git-branch ${GIT_BRANCH} --github-script ${env.serverGithubScriptPath} --mode p2p --workspace ${WORKSPACE}"
+                    }
+                }
+            }
+        }
+        stage('runAndroidCICase'){
+            agent{
+                node {
+                    label "${env.androidRunCaseServer}"
+                    customWorkspace "${env.androidSourcePath}"
+                }
+            }
+            steps {
+                sh "docker run -v /dev/bus/usb:/dev/bus/usb -v ${env.androidCILogPath}:/home/logs --network host --privileged --rm  ${env.androidSdkImage} \
+                python ${androidSdkTestScript} --git-branch master --git-change-id null --git-commit ${GIT_COMMIT} --owner open-webrtc-toolkit --repo owt-client-native \
+                --remote-dependence-host ${env.remoteDependenceHost} --remote-dependence-host-username ${env.remoteDependenceUserName} \
+                --remote-dependence-host-password ${env.remoteDependencePassword} --remote-dependence-path ${env.remoteDependeceDebugPath}  \
+                --instrumentation ${env.androidSdkCaselistPath} --unit --p2p-server http://${env.p2pServer}:8095 --conference-server-http http://${env.mcuServer}:3001"
```

Comment:
Don't hardcode port e.g 8095 and 3001 , update port don't need update jenkinfile

Reply:
done